tests/publisher_test.py

```diff
@@ -404,3 +404,90 @@ def testUsesUnionOfMetaKeys(self):
     rows = list(reader)
     self.assertEqual(['key1', 'key3'], reader.fieldnames[-2:])
     self.assertEqual(3, len(rows))
+
+
+  class InfluxDBPublisherTestCase(unittest.TestCase):
+    def setUp(self):
+      self.db_name = 'test_db'
+      self.db_uri = 'test'
+      self.mock_db = publisher.InfluxDBPublisher(self.db_uri, self.db_name)
```

Comment:
Since this is the actual instance of a publisher, I think `mock` shouldn't be included in the variable name, since it can be confused for a mock object.